build.sbt

```diff
@@ -651,7 +651,7 @@ lazy val json =
       description := "JSON parser"
     )
     .jsSettings(jsBuildSettings)
-    .dependsOn(log, airframeSpec % "test")
+    .dependsOn(log, msgpack, airframeSpec % "test")
```

Comment:
Is there no problem with this as dependsOn?

Reply:
It's OK for now. I'll follow-up this issue in #574